mainnet-canisters.json

```diff
@@ -36,8 +36,8 @@
     "sha256": "fd25a4e2e283b498c3be1aaf63cc9b2726264d78a12b12f43ad453ceeb575e7c"
   },
   "governance": {
-    "rev": "f5a63fd41933eee24ab3a6c71e43c67c1048e0db",
-    "sha256": "c7cd011d711edadc024d2673c090cea17688eca1748648d04c0c0afbdaebdbdc"
+    "rev": "0f35ac817bdf3b44680111fc90ce5d6e3bc79be1",
```

Comment:
This seems wrong, it should be the same rev as in the other two cases: https://dashboard.internetcomputer.org/proposal/134340